pkg/util/util.go

```diff
@@ -29,3 +31,26 @@ func IsGPUVMI(vmi *v1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+
+func isIpv6Disabled() bool {
```

Comment:
I think it's more readable when positive logic is used - i.e. I would have a `isIpv6Enabled` function instead.

This is subjective though.

Reply:
just because this is the name of the var, and can be mistaken to think that ipv6 is used if 
the word enabled appear, i prefer this method if it ok (as you said subjective),
else i would change to positive logic